includes/getting-started.php

```diff
@@ -65,7 +42,7 @@
 			target="_blank">SemPress</a> is the only theme in the WordPress repository that is fully
 			microformats2 compliant. <a href="http://wordpress.org/themes/independent-publisher/"
 			target="_blank">Independent Publisher</a> has been updated to include basic microformats2 and
-			webmention display support. In practice, most themes will work relatively well out of the box, though there can be some minor display issues.', 'indieweb'
+			webmention display support. In practice, most themes will work relatively well out of the box, though there can be some minor display issues. If you are interested in receiving help in converting your theme to support Microformats 2, the community will be happy to assist.', 'indieweb'
```

Comment:
Perhaps a link to the indieweb.org/discuss page for indieweb-wordpress on the "community will be happy to assist"?